src/test/adapter-factory/transactions.ts

```
@@ -0,0 +1,40 @@
+				return;
+			}
+
+			const user1 = await generate("user");
+			const user2 = await generate("user");
+			await expect(
+				adapter.transaction(async (tx) => {
+					await tx.create({ model: "user", data: user1, forceAllowId: true });
+					const users = await tx.findMany({ model: "user" });
+					expect(users).toHaveLength(1);
+					throw new Error("Simulated failure");
+					await tx.create({ model: "user", data: user2, forceAllowId: true });
```

Biome (2.4.4)

[error] This code will never be reached ...

(lint/correctness/noUnreachable)